jbpm-bpmn2/src/test/resources/BPMN2-DataInputAssociations-Pojo-Complete.bpmn2

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?> 
+<definitions id="Definition"
+             targetNamespace=""
+             typeLanguage="http://www.java.com/javaTypes"
+             expressionLanguage="http://www.mvel.org/2.0"
+             xmlns="http://www.omg.org/spec/BPMN/20100524/MODEL"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd"
+             xmlns:g="http://www.jboss.org/drools/flow/gpd"
+             xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
+             xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
+             xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
+             xmlns:tns="http://www.jboss.org/drools">
+
+  <itemDefinition id="_instanceMetadataItem" structureRef="org.jbpm.bpmn2.DataTest$Person" />
```

Comment:
This value including "$" is not valid as QName:
_The 'structureRef' attribute is invalid - The value 'org.jbpm.bpmn2.DataTest$Person' is invalid according to its datatype 'http://www.w3.org/2001/XMLSchema:QName' - The '$' character, hexadecimal value 0x24, cannot be included in a name._

which it's shown in the parser as an error:
```
ERROR Node Info: id:_instanceMetadataItem name: 
Parser message: (null: 15, 94): cvc-datatype-valid.1.2.1: 'org.jbpm.bpmn2.DataTest$Person' is not a valid value for 'QName'.
```

though dollar-sign ($) is the way that mvel 2.0 uses to access nested classes

Reply:
This $ is the name of the inner class, since this is test, I think we can live with it (I will prefer to not make this class available in a different file since it only applies to this test)